configs/dla/faster_rcnn_dla_60_fpn_1x_coco.py

```diff
@@ -0,0 +1,20 @@
+_base_ = '../faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py'
+
+model = dict(
+    pretrained='open-mmlab://dla60',
+    backbone=dict(
+        type='DLANet',
+        depth=60,
+        num_stages=4,
+        out_indices=(0, 1, 2, 3),
+        frozen_stages=1,
+        norm_cfg=dict(type='BN', requires_grad=True),
+        norm_eval=True,
+        style='pytorch'),
+    neck=dict(
+        type='FPN',
+        in_channels=[128, 256, 512, 1024],
+        out_channels=256,
+        num_outs=5))
+
+find_unused_parameters = True
```

Comment:
This should be unnecessary as only the backbone and neck are changed. Otherwise, we should clean the code of implementation.

Reply:
@ZwwWayne There exist two arguments in the forward function of BottleNeck and BasicBlock. If I delete this line, the procedure will throw errors.

Reply:
There existed unused parameters which were found difficultly, so I added this line to make the procedure run normally. Now I have solved this problem